packages/tap-runner/test/integration/tap-test-runner.it.spec.ts

```diff
@@ -208,4 +208,41 @@ describe('tap-runner integration', () => {
       assertions.expectCompleted(run);
     });
   });
+
+  describe('Running on a ava project', () => {
+    beforeEach(async () => {
+      sandbox = new TempTestDirectorySandbox('ava');
+      await sandbox.init();
+
+      options.tap = tapRunnerOptions({
+        nodeArgs: [getAvaLocation(), '--tap'],
+      });
+      sut = testInjector.injector.injectFunction(createTapTestRunnerFactory('__stryker2__'));
+      await sut.init();
+    });
+
+    it('should be able to run', async function () {
+      // Test fails if debugger is used, because ava doesn't support debugging and tap output simultaneously.
+      // See: https://github.com/avajs/ava/blob/568fe40c987dd6c593dfbcf4144d1d1627955d46/lib/cli.js#L307
+      const { default: inspector } = await import('node:inspector');
+
+      if (inspector.url() !== undefined) {
+        this.skip();
+      } else {
+        // Act
+        const run = await sut.dryRun(factory.dryRunOptions());
+
+        // Assert
+        assertions.expectCompleted(run);
+      }
+    });
+
+    function getAvaLocation() {
+      if (os.platform() === 'win32') {
+        return `${sandbox.tmpDir}/../../../node_modules/ava/entrypoints/cli.mjs`;
```

Comment:
```suggestion
        return path.resolve(sandbox.tmpDir, '..', '..', '..', 'node_modules', 'ava', 'entrypoints', 'cli.mjs');
```

Maybe a small improvement?